src/Microsoft.ML/Runtime/EntryPoints/MacroUtils.cs

```diff
@@ -30,9 +30,8 @@ public sealed class EvaluatorSettings
         {
             public string LabelColumn { get; set; }
             public string NameColumn { get; set; }
-            public string ScoreColumn { get; set; }
-            public string[] StratColumn { get; set; }
```

Comment:
just curios, why?
don't we use it?

Reply:
We don't. This class is used only when instantiating evaluator entry points in macros, where the score column always has the default name. The stratification option is not exposed there either.

---
In reply to: [192859511](https://github.com/dotnet/machinelearning/pull/291#discussion_r192859511) [](ancestors = 192859511)